sentry-android-core/src/main/java/io/sentry/android/core/SentryPerformanceProvider.java

```diff
@@ -102,7 +101,16 @@ public void onActivityCreated(@NotNull Activity activity, @Nullable Bundle saved
   public void onActivityStarted(@NotNull Activity activity) {}
 
   @Override
-  public void onActivityResumed(@NotNull Activity activity) {}
+  public void onActivityResumed(@NotNull Activity activity) {
+    if (!firstActivityResumed) {
```

Comment:
`m`: hmm, do we still need to check for `foregroundImportance` here actually?

Reply:
The `foregroundImportance` check is still done inside the `ActivityLifecycleIntegration`, as it's also used for creating the transaction as well.
So, yes the end-time in `AppStartState` is still set, but it's ignored in the `ActivityLifecycleIntegration`, so I guess it's fine.

Reply:
`foregroundImportance` is about if the App received a notification and no screen was shown or something similar, so the app start is inaccurate because we end the app start only when there's a visible screen, I'd say unless you fix this problem using another technique, this should stay.